docs/how-to-guides/performing-cluster-maintenance.md

```diff
@@ -0,0 +1,239 @@
+# Perform cluster maintenance
+
+MicroCeph provides a simple and consistent workflow to support cluster maintenance activity.
+
+## Prerequisite
+
+Cluster maintenance requires extra redundancy in ceph services, make sure you have at least 4 units of microceph and
+that all units have ceph monitor enabled.
+
+For example, one possible 4 nodes MicroCeph cluster can be like:
```

Comment:
[* suggested rewording *] : A four-node MicroCeph cluster would look something similar to this: